todoman/ui.py

```diff
@@ -251,8 +259,19 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        :param datetime.datetime date: a datetime object
+        Returns date in the following format:
+        * if date == today or tomorrow: "Today" or "Tomorrow"
+        * else: return a string representing that date
+        * if no date is supplied, it returns empty_date
+        """
+        assert isinstance(date, datetime)
```

Comment:
This needs to be moved one line down:
```python
if date:
    assert(...
```

Otherwise it fails when date is `None` (in which case we'd return `empty_date`).